tests/testthat.R

```diff
@@ -1,18 +1,4 @@
 library(testthat)
 library(rsconnect)
 
-run_tests <- function() {
```

Comment:
This is no longer needed because we do it per test, where necessary. And that also means that you can run the tests interactively without messing with your personal server/account config.

Reply:
It should be picked up by CI, since we did some work to ensure that check is run on GHA.